.bin/srclib-csharp

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+BIN_DIR="`dirname \"$BASH_SOURCE\"`"
+ROOT_DIR="`dirname \"$BIN_DIR\"`"
+PROJ_DIR="$ROOT_DIR/Srclib.Nuget"
+#echo "$PROJ_DIR"
+
+#echo dnx -p "$PROJ_DIR/project.json" run $*
+dnx -p "$PROJ_DIR" run $*
```

Comment:
I'm still not sure this is a convenient way. As far as I can see, none of the toolchains has dep target in their Makefiles

https://github.com/sourcegraph/srclib-java/blob/master/Makefile

https://github.com/sourcegraph/srclib-python/blob/master/Makefile

https://github.com/sourcegraph/srclib-go/blob/master/Makefile

https://github.com/sourcegraph/srclib-basic/blob/master/Makefile

Java toolchain doesn't try to install jdk and jre, go toolchain doesn't try to install go compiler and so on. Similarly, csharp toolchain shouldn't try to set up dnx environment. If we add installation to Makefile, then we will have to invoke make with super user privileges:

sudo make dep

If srclib/cli/toolchain_cmd.go invokes 'make dep', then we will have to run srclib with super user privileges, like:

sudo srclib toolchain install csharp